kpi/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> Daniel Reis <dreis.pt@hotmail.com>
+# Copyright <YEAR(S)> Glen Dromgoole <gdromgoole@tier1engineering.com>
+# Copyright <YEAR(S)> Loic Lacroix <loic.lacroix@savoirfairelinux.com>
+# Copyright <YEAR(S)> Sandy Carter <sandy.carter@savoirfairelinux.com>
+# Copyright <YEAR(S)>Gervais Naoussi <gervaisnaoussi@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
IMO copyright is attributed to original authors only, and contributors should not be included on authorship and copyright attribution.